bash_completions.go

```diff
@@ -88,6 +88,11 @@ __handle_reply()
     fi
     COMPREPLY=( $(compgen -W "${completions[*]}" -- "$cur") )
 
+    if [[ ${#completions[@]} -eq 0 ]]; then
+    	__debug "${FUNCNAME}: no known completions, falling back to _filedir"
+    	_filedir
+    fi
+
```

Comment:
also, easy way to get that in your code would be just add _filedir to the end of your __custom_func....